wait-for.bats

```diff
@@ -13,6 +13,12 @@
   [ "$output" != "success" ]
 }
 
+@test "support version option" {
+  run ./wait-for -v
+
+  [ "$output" = "2.1.3" ]
```

Comment:
Maybe here we could check if the output is in sync with the `version` stated in `package.json` 🤷 

Just to make sure that our version-bump logic succeeded.